src/main/java/de/zalando/ep/zalenium/servlet/renderer/CloudProxyHtmlRenderer.java

```diff
@@ -87,13 +87,13 @@ private String getSingleSlotHtml(TestSlot s) {
         TestSession session = s.getSession();
         String icon = "";
         if (proxy instanceof TestingBotRemoteProxy) {
-            icon = "/grid/admin/ZaleniumResourceServlet/images/testingbot.png";
+            icon = "/grid/resources/images/testingbot.png";
```

Comment:
Maybe just straight `/resources` pretty sure the hub doesn’t use it

Reply:
Maybe let’s just merge and when we add zalenium Prometheus metrics. I’ll move the resources somewhere else now that I know what uses them.